### PR TITLE
Defining a subset of methods as in PyTango's DeviceProxy

### PR DESCRIPTION
**Pull request by bohrium272**

### CODE REVIEW

File: MOCC/src/configurations/Comm-Layer/device_proxy.py [hunk 1]
```diff
@@ -1,38 +1,160 @@
 import requests
+from config_manager import ConfigManager
```
Comment (mtambos) [inline MOCC/src/configurations/Comm-Layer/device_proxy.py]: The config manager is not a library (yet). It is a REST server, so you need to make a REST call to get the dev_name <-> IP:port mapping

File: MOCC/src/configurations/Comm-Layer/device_proxy.py [hunk 2]
```diff
@@ -1,38 +1,160 @@
...
     def bind_functions(self):
         functions_dict = self.get_function_list()
         functions = functions_dict['functions']
 
         for name in functions:
             self.add_fn(name)
+
+    # Returns n previous commands
+    def black_box(self, n):
+        raise NotImplementedError
```
Comment (mtambos) [inline MOCC/src/configurations/Comm-Layer/device_proxy.py]: you forgot the parentheses after "...Error"

**Review by mtambos — CHANGES_REQUESTED** [review]

File: MOCC/src/configurations/Comm-Layer/device_proxy.py (hunk 1 above)
Comment (bohrium272) [reply MOCC/src/configurations/Comm-Layer/device_proxy.py]: Where do I get the address for the REST server, or is it fixed?

File: MOCC/src/configurations/Comm-Layer/device_proxy.py [hunk 3]
```diff
@@ -2,37 +2,160 @@
...
     def bind_functions(self):
         functions_dict = self.get_function_list()
         functions = functions_dict['functions']
 
         for name in functions:
             self.add_fn(name)
+
+    # Returns n previous commands
+    def black_box(self, n):
+        raise (NotImplementedError)
```
Comment (mtambos) [inline MOCC/src/configurations/Comm-Layer/device_proxy.py]: this (and all following) should be `raise NotImplementedError()`

**Review by mtambos — APPROVED** [review]

**Pull request merged**